web-server/routes/skills.js

```diff
@@ -31,9 +31,11 @@ module.exports = app => {
     queries.getChecksForUserAndLabels({userId, labels})
       .then(checks => {
         const checked = !!checks[skill.id]
-        response.render('skills/show', {skill, checked, title: skill.name})
```

Comment:
revert you changes to the helpers and make a new helper called `renderMarkdownFileToHTML`

```js
const markdownHTML = response.renderMarkdownFileToHTML(`/skills/${skill.id}/README.md`)
response.render('skills/show', {markdownHTML, skill, checked, title: skill.name})
```

Reply:
@deadlyicon I can't imagine doing this without being async. Are you okay with:
```
response.renderMarkdownFileToHTML(`/skills/${skill.id}/README.md`)
.then( markdownHTML => {
  response.render('skills/show', {markdownHTML, skill, checked, title: skill.name})
})
```

Reply:
of course! it has to be asyc because we read files async :)